src/dicts.py

```diff
@@ -6,7 +6,18 @@
 #
 # Make up three entries of various values.
 
-waypoints = [
-]
+waypoints = {
```

Comment:
You've created a single dictionary with three keys, but the assignment is looking for an array of three such dictionaries (with lat/lon/name properties). This will look nearly identical to JSON.